spec/api-power-monitor-spec.js

```diff
@@ -85,5 +85,34 @@ const skip = process.platform !== 'linux' || !process.env.DBUS_SYSTEM_BUS_ADDRES
         })
       })
     })
+
+    describe('when a listener is added to shutdown event', () => {
+      before(async () => {
+        const calls = await getCalls()
+        assert.equal(calls.length, 2)
+        powerMonitor.once('shutdown', () => { })
```

Comment:
Why all of it should be done in a `before` hook?
I don't see how it prepares an environment or context for tests.

Reply:
Normally I try to structure BDD tests like this:

- `describe` describes the context
- `beforeEach` sets up the context
- `it` describes/verifies an expectation about the context

In this case the context is "when a listener is added to shutdown event" which is configured in the `before` callback.  The `it` call describes/verifies the "should call Inhibit to delay shutdown" expectation. I also added an assertion in the `before` callback to make it explicit that `getCalls` returns two calls prior to adding the listener.

Reply:
ok